vme/src/unit_data.h

```diff
@@ -85,6 +112,22 @@ class unit_data : public basedestruct
     file_index_type *getFileIndex();
     const file_index_type *getFileIndex() const;
     void setFileIndex(file_index_type *value);
+    [[nodiscard]] const char *getFileIndexZoneName() const
+    {
+        if (m_fi)
+        {
+            return m_fi->getZone()->getName();
+        }
+        return "NO-ZONE";
```

Comment:
I can't recall if "-" is a valid character in a FI name / zone name. Space definitely isn't valid. Thinking it's a good idea to have a name here that isn't valid.